IUFRO-HQ.csl

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" et-al-min="3" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="en-GB">
```

Comment:
I'd generally avoid setting `et-al-min` at the style level, especially since it looks like you're overriding it for every occurrence.

Reply:
I removed 'et-al-min' at the style level